### DISCUSSION TITLE
大佬们都是用什么显卡啊

### DISCUSSION THREAD

**Discussion by KaranaLJX** [wenda-LLM/wenda#206] [Q&A] [answered]
遇到报错了，torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 32.00 MiB (GPU 0; 2.00 GiB total capacity; 1.71 GiB already allocated; 0 bytes free; 1.72 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

**Comment by l15y** [comment, marked as answer]
至少6g显存能好用

**Comment by kangyiwen** [comment]
我的是：Nvidia GeForce GTX 1660 SUPER ( 6 GB / 七彩虹 )

**Comment by lizi1314520** [comment]
M40  才450  24G显存，可以入手，我的P40  1088

**Reply by lbch** [reply]
m40的cuda版本可有点太旧了

**Comment by wu5shen** [comment]
2060s完美运行6b，RWKV模型内存16g基本上运行不了

**Comment by lizi1314520** [comment]
P40&nbsp; &nbsp;24g，爽，打游戏用1060 6G




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "l15y/wenda"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月8日(星期一) 晚上11:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [l15y/wenda] 大佬们都是用什么显卡啊 (Discussion #206)





 
2060s完美运行6b，RWKV模型内存16g基本上运行不了

**Comment by serfan** [comment]
48G内存 3060 6G 勉强可以跑一下。

**Comment by jellzone** [comment]
4090

**Comment by zhuang-maowei** [comment]
A6000 * 2
